cookbooks/aws-parallelcluster-platform/templates/nvidia-imex/nvidia-imex.service.erb

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=NVIDIA IMEX service
+After=network-online.target
+Requires=network-online.target
+
+[Service]
+Environment="KRB5_CLIENT_KTNAME=/etc/krb5.keytab"
+User=root
+PrivateTmp=false
+Type=forking
+TimeoutStartSec=infinity
+
+ExecStart=/usr/bin/nvidia-imex -c <%= node['cluster']['nvidia']['imex']['shared_dir'] %>/config.cfg
```

Comment:
We reference the path to nvidia-imex binary and config file in different places. What about injecting the variables directly?

`ExecStart=<%= node['cluster']['nvidia']['imex']['binary'] %>` -c <%= node['cluster']['nvidia']['imex']['config'] %>`

Reply:
So you want a variable for the binary? Why? we dont decide this path